stubs/netifaces/netifaces.pyi

```diff
@@ -0,0 +1,74 @@
+from typing import Dict, List, Tuple, Union, Literal
```

Comment:
None of these – except `Literal` should be imported. Please use built-in types (`dict`, `list`, `tuple`) and new-style union syntax `X | Y`. (I thought our tests would complain about this, but they didn't ...)

Reply:
Done.